addons/survey/migrations/13.0.3.0/pre-migration.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_model_renames = [
+    ('survey.mail.compose.message', 'survey.invite'),
+]
+
+_table_renames = [
+    ('survey_mail_compose_message', 'survey_invite'),
+]
+
+_field_renames = [
+    ('survey.label', 'survey_label', 'quizz_mark', 'answer_score'),
+    ('survey.question', 'survey_question', 'type', 'question_type'),
+    ('survey.survey', 'survey_survey', 'email_template_id', 'certification_mail_template_id'),
+    ('survey.user_input', 'survey_user_input', 'type', 'input_type'),
+    ('survey.user_input', 'survey_user_input', 'date_create', 'start_datetime'),
+    ('survey.user_input_line', 'survey_user_input_line', 'date_create', 'value_datetime'),
+    ('survey.user_input_line', 'survey_user_input_line', 'quizz_mark', 'answer_score'),
+]
+
+_field_adds = [
+    ("title", "survey.question", "survey_question", "char", False, "survey"),
+    ("survey_id", "survey.question", "survey_question", "many2one", False, "survey"),
+    ("is_page", "survey.question", "survey_question", "boolean", False, "survey"),
+]
+
+_xmlid_renames = [
+    # ir.model.access
+    ('survey.access_survey_label_manager', 'survey.access_survey_label_survey_manager'),
+    ('survey.access_survey_label_public', 'survey.access_survey_label_all'),
+    ('survey.access_survey_manager', 'survey.access_survey_survey_manager'),
+    ('survey.access_survey_public', 'survey.access_survey_all'),
+    ('survey.access_survey_question_manager', 'survey.access_survey_question_survey_manager'),
+    ('survey.access_survey_question_public', 'survey.access_survey_question_all'),
+    ('survey.access_survey_user_input_line_manager', 'survey.access_survey_user_input_line_survey_manager'),
+    ('survey.access_survey_user_input_line_public', 'survey.access_survey_user_input_line_all'),
+    ('survey.access_survey_user_input_manager', 'survey.access_survey_user_input_survey_manager'),
+    ('survey.access_survey_user_input_public', 'survey.access_survey_user_input_all'),
+    # ir.rule
+    ('survey.survey_input_manager_access', 'survey.survey_user_input_rule_survey_manager'),
+    ('survey.survey_input_public_access', 'survey.survey_user_input_rule_survey_user_read'),
+    ('survey.survey_input_users_access', 'survey.survey_user_input_rule_survey_user_cw'),
+    ('survey.survey_manager_access', 'survey.survey_survey_rule_survey_manager'),
+    ('survey.survey_users_access', 'survey.survey_survey_rule_survey_user_read'),
+    # mail.template
+    ('survey.email_template_survey', 'survey.mail_template_user_input_invite'),
+]
+
+
+def move_survey_page_to_survey_question(env):
+    openupgrade.add_fields(env, _field_adds)
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE survey_question sq
+        SET title = sp.title AND survey_id = sp.survey_id AND is_page = TRUE
```

Comment:
incorrect syntax use `,` not `AND` to avoid

```
psycopg2.ProgrammingError: argument of AND must be type boolean, not type character varying
LINE 3:         SET title = sp.title AND survey_id = sp.survey_id AN...
```